modules/helfi_node_survey/config/install/core.entity_form_display.node.survey.default.yml

```diff
@@ -0,0 +1,152 @@
+uuid: 2dea284e-6cc0-4e34-9d2f-37baa7317502
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.survey.body
+    - field.field.node.survey.field_survey_content_pages
+    - field.field.node.survey.field_survey_link
+    - node.type.survey
+  module:
+    - hdbt_admin_tools
+    - link
+    - path
+    - publication_date
+    - scheduler
+    - select2
+    - text
+id: node.survey.default
+targetEntityType: node
+bundle: survey
+mode: default
+content:
+  body:
+    type: text_textarea_with_summary
+    weight: 18
+    region: content
+    settings:
+      rows: 9
+      summary_rows: 3
+      placeholder: ''
+      show_summary: false
+    third_party_settings: {  }
+  created:
+    type: datetime_timestamp
+    weight: 3
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  field_survey_content_pages:
+    type: select2_entity_reference
+    weight: 14
+    region: content
+    settings:
+      width: 100%
+      autocomplete: true
+      match_operator: CONTAINS
+      match_limit: 20
+    third_party_settings: {  }
+  field_survey_link:
+    type: link_default
+    weight: 19
+    region: content
+    settings:
+      placeholder_url: ''
+      placeholder_title: ''
+    third_party_settings: {  }
+  field_survey_service_pages:
```

Comment:
The service and unit page settings didn't get installed on any instance.

Reply:
As discussed, I'll remove these options.